crm_phonecall/views/res_partner_view.xml

```diff
@@ -8,14 +8,14 @@
         <field name="groups_id" eval="[(4, ref('sales_team.group_sale_manager')), (4, ref('base.group_partner_manager'))]"/>
         <field eval="18" name="priority"/>
         <field name="arch" type="xml">
-            <xpath expr="//div[@name='button_box']" position="inside">
```

Comment:
Please don't change this. It should be inside the div, not before another button

Reply:
I did this because otherwise the button placed after the "Active" button. And it doesn't look good. All other buttons in odoo are added like that.

ie: the crm Opportunities button:
![image](https://user-images.githubusercontent.com/1914185/63175634-b34d2f80-c044-11e9-8f27-6969e527c848.png)


Changing it like this have the same effect, only it keeps the toggle_active button at the end, which is the standard in odoo

Reply:
OK, thanks for the extra explanations. Seems fair. In v13 there's no more active smart-button, so we will need to change it again, hehe.